package.json

```diff
@@ -169,13 +169,11 @@
     "mock-fs": "^4.13.0",
     "portfinder": "^1.0.28",
     "pusher-js-mock": "^0.3.3",
-    "react-hot-loader": "^4.12.21",
     "react-native-clean-project": "^4.0.0-alpha4.0",
-    "react-native-flipper": "^0.146.1",
+    "react-native-flipper": "https://gitpkg.now.sh/facebook/flipper/react-native/react-native-flipper?9cacc9b59402550eae866e0e81e5f0c2f8203e6b",
```

Comment:
Hack: This was merged a few hours before I made this PR, I assume it will land soon-ish.

Reply:
According to their GitHub, this should land in `0.160.0` but I haven't seen it hit the npm registry yet.